Sources/TimelaneCore/TimelaneCore.swift

```diff
@@ -18,6 +18,11 @@ public class Timelane {
             return OSLog(subsystem: "tools.timelane.subscriptions", category: "DynamicStackTracing")
         }
     }()
+
+    enum LaneType: Int, CaseIterable {
```

Comment:
@donnywals shouldn't this one be public?

Reply:
You'd think I wouldn't forget this kind of stuff considering I work on frameworks full-time.. 😬